An attribute must have a value (#36450)
mdn/content: files/en-us/glossary/boolean/html/index.md

{{GlossarySidebar}}

A **boolean attribute** in {{Glossary("HTML")}} is an {{glossary("attribute")}} that represents `true` or `false` values. If an HTML tag contains a boolean attribute - no matter the value of that attribute - the attribute is set to `true` on that element. If an HTML tag does not contain the attribute, the attribute is set to `false`.
A **boolean attribute** in {{Glossary("HTML")}} is an {{glossary("attribute")}} that represents `true` or `false` values. If an HTML tag contains a boolean attribute --- no matter the value of that attribute --- the attribute is set to `true` on that element. If an HTML tag does not contain the attribute, the attribute is set to `false`.

If the attribute is present, it can have one of the following values:

> - no value at all, e.g. `attribute`

Reviewer: This change isn’t correct. See the **Empty attribute syntax** HTML spec section you cited in the PR description

Author: The HTML spec page says:

> Just the [attribute name](https://html.spec.whatwg.org/multipage/syntax.html#syntax-attribute-name). The value is implicitly the empty string.

It's not no value.

Reviewer: > The HTML spec page says:
> 
> > Just the [attribute name](https://html.spec.whatwg.org/multipage/syntax.html#syntax-attribute-name). The value is implicitly the empty string.
> 
> It's not no value.

Yeah like I said, a change to something like that same wording would make everything clear, and be more accurate. But not completely dropping any statement that the attribute can be provided in the HTML markup source with just the attribute name, and no explicit value stated.

{{GlossarySidebar}}

> A **boolean attribute** in {{Glossary("HTML")}} is an {{glossary("attribute")}} that represents `true` or `false` values. If an HTML tag contains a boolean attribute --- no matter the value of that attribute --- the attribute is set to `true` on that element. If an HTML tag does not contain the attribute, the attribute is set to `false`.

Reviewer: ```suggestion
A **boolean attribute** in {{Glossary("HTML")}} is an {{glossary("attribute")}} that represents `true` or `false` values. If an HTML tag contains a boolean attribute — no matter the value of that attribute — the attribute is set to `true` on that element. If an HTML tag does not contain the attribute, the attribute is set to `false`.
```

If the attribute is present, it can have one of the following forms:

> - the attribute name alone, e.g., `attribute`, meaning it has an implicit empty string value

Reviewer: ```suggestion
- the attribute name alone; e.g., `attribute`, meaning it's implicit value is the empty string
```
